client/src/config/ga.js

```diff
@@ -9,5 +9,5 @@ export const events = {
     `${change} ${filter}`,
     window.location
   ],
-  download: (type, project, sample) => ['Download', type, sample || project]
+  download: (project, sample) => ['Download', 'Clicked', sample || project]
```

Comment:
```suggestion
  download: (project, sample) => ['Download', 'Downloaded', sample || project]
```

Since this isn't a click let's just rename the action to "download"